tests/ui/perfherder/compare_table_test.jsx

```diff
@@ -130,3 +174,70 @@ test('text input filter results should differ when filter button(s) are selected
   expect(result1).not.toBeInTheDocument();
   expect(result2).not.toBeInTheDocument();
 });
+
+test('retrigger buttons should be present if user is logged in', async () => {
+  const { queryAllByTitle } = compareTableControls(true);
+  const retriggerButtons = queryAllByTitle(
+    compareTableText.retriggerButtonTitle,
+  );
+  expect(retriggerButtons).toHaveLength(2);
+});
+
+test('retrigger buttons should not exist if user is not logged in', async () => {
+  const { queryAllByTitle } = compareTableControls(false);
+  const retriggerButtons = queryAllByTitle(
+    compareTableText.retriggerButtonTitle,
+  );
+  expect(retriggerButtons).toHaveLength(0);
+});
```

Comment:
You could also combine the first two tests by testing it first with false, then mimic a logging in action.